README.md

```diff
@@ -1,73 +1,85 @@
+We hold these truths to be self-evident:
+that all men are created equal,
+that they are endowed by their Creator with certain unalienable Rights,
+that among these are Life, Liberty and the pursuit of Happiness,
+that to secure these rights, governments are instituted among men, deriving their just powers from the consent of the governed,
+that whenever any form of government becomes destructive of these ends, it is the right of the people to alter or to abolish it, & to institute new government, laying its foundation on such principles & organizing its powers in such form, as to them shall seem most likely to effect their Safety and Happiness.
+Prudence, indeed, will dictate that Governments long established should not be changed for light and transient causes,
+and accordingly all experience hath shewn that mankind are more disposed to suffer, while evils are sufferable than to right themselves by abolishing the forms to which they are accustomed.
+But when a long train of abuses & usurpations, pursuing invariably the same Object evinces a design to reduce them under absolute Despotism, it is their right, it is their duty, to throw off such Government, and to provide new Guards for their future security.
+Such has been the patient sufferance of these Colonies; and such is now the necessity which constrains them to alter their former Systems of Government.
+The history of the present King of Great Britain is a history of repeated injuries and usurpations, all having in direct object the establishment of an absolute Tyranny over these States.
+To prove this, let Facts be submitted to a candid world.
 
-HE has refused his assent to laws the most wholesome & necessary for the public good.
+He has refused his assent to laws, the most wholesome & necessary for the public good.
```

Comment:
Alright, alright.

On Mon, Jul 4, 2016 at 10:58 PM, Andrew Meyer <notifications@github.com>
wrote:

> In README.md
> <https://github.com/usgov/forget-the-king/pull/1#discussion_r69488490>:
>
> >
> > -HE has refused his assent to laws the most wholesome & necessary for the public good.
> > +He has refused his assent to laws, the most wholesome & necessary for the public good.
>
> Pretty sure the "present King of Great Britain" at the time was a King,
> not a Queen...
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/usgov/forget-the-king/pull/1/files/966a6f3345f8e4c60f5910a7b37cf0d3f63538e5#r69488490>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAhPPEK20wXSP4t0e8QXvZU4dYd1u-S5ks5qSWX6gaJpZM4JDeFy>
> .
>